queue_services/entity-emailer/src/entity_emailer/message_tracker/tracker.py

```diff
@@ -171,16 +180,30 @@ def start_tracking_message(message_context_properties: dict, email_msg: dict, ex
     return create_processing_message(message_context_properties, email_msg)
 
 
-def complete_tracking_message(tracker_msg: MessageProcessing):
+def complete_tracking_message(tracker_msg: MessageProcessing, email_msg):
     """Update existing message state to COMPLETED."""
     update_message_status_to_complete(tracker_msg)
 
+    # Update corresponding furnishings entry as PROCESSED
+    etype = email_msg.get('type', None)
+    if etype and etype == 'bc.registry.dissolution':
+        furnishing_id = email_msg['data']['furnishing']['furnishingId']
+        ar_overdue_stage_1_notification.update_furnishing_status(furnishing_id,
+                                                                 Furnishing.FurnishingStatus.PROCESSED)
```

Comment:
this should not be part of tracker code.

it should take place in the processor

Reply:
Updated